docs/variants/protectli_ptx01/releases.md

```diff
@@ -15,3 +15,6 @@ Test results for this platform can be found
 [here](https://docs.google.com/spreadsheets/d/1wI0qBSLdaluayYsm_lIa9iJ9LnPnCOZ9eNOyrKSc-j4/edit#gid=1316498194).
 
 [newsletter]: https://newsletter.3mdeb.com/subscription/n2EpSxtqL
+
+<!--Empty pixel to avoid orphaned pages when overview is hidden-->
+[![empty-pixel](../../images/empty_pixel.png)](overview.md)
```

Comment:
Not sure i understand what is it for... Why other files do not need it?

Reply:
@miczyg1 We do not have other local links to `overview.md` when it is hidden and for this reason it becomes an orphaned page. Adding an empty pixel with a link to `overview.md` fixes this problem.